app/controllers/public/sessions.js

```diff
@@ -0,0 +1,28 @@
+import Ember from 'ember';
+import moment from 'moment';
+import { orderBy, pick, groupBy, keyBy, mapValues, last } from 'lodash';
+
+const { Controller, computed } = Ember;
+
+export default Controller.extend({
+  sessions: [{ shortAbstract: 'An introduction to the event', id: '1', title: 'Welcome to FOSSASIA', startAt: moment(new Date()).format('h:mm a / DD-MM-YYYY'), track: { color: 'green', fontColor: 'green', id: 1, name: 'Track 1' }, microlocation: { id: 2, name: 'Room 1' }, speakers: [{ shortBiography: 'Works for ORG 1', id: 0, city: 'Delhi', name: 'Arnold Singh', speakingExperience: 'GSOC 2015', organisation: 'ORG 1', longBiography: '', photo: { 'iconImageUrl': '' } }] },
```

Comment:
Again ... like I said `startAt` and `endAt` will NOT be a string. It will be a date object. You need to use [ember-moment](https://github.com/stefanpenner/ember-moment) template helpers to display it how you like.